packages/components/package.json

```diff
@@ -19,8 +19,8 @@
     "snarkdown": "^2.0.0"
   },
   "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup --watch -c",
+    "build": "yarn --silent && tsc --build",
+    "watch": "yarn --silent && tsc --build --watch",
```

Comment:
rollup was removed. Here the scripts are purely to test the typings with tsc. If we want to build a bundle we can use `parcel build`, but considering the components package is a npm module I don't think we need to bundle anything. Simply distributing what tsc produces should work fine.